packages/framework/src/Resources/config/services.yml

```diff
@@ -1014,12 +1017,3 @@ services:
             - method: setEntityManager
 
     Shopsys\FrameworkBundle\Model\Product\Search\ProductElasticsearchConverter: ~
-
-    Shopsys\FrameworkBundle\Command\CheckRedisCommand:
```

Comment:
will this be also part of upgrade note, in the case it was overriden in shopBundle ???

Reply:
Both changes will be a part of the upgrade note. If it was overriden, there are instances of Redis passed directly, which triggers the deprecation error.